### PR TITLE
Fixed "${TEST_INDEX} is initialized to 'null' for video recording path pattern"  (close #3455)

### LINKED ISSUE #3455: ${TEST_INDEX} is initialized to 'null' for video recording path pattern
### What is your Test Scenario?

Running a simple test with a video recording feature enabled. 

### What is the Current behavior?

The 'null' folder name is created for the ${TEST_INDEX} variable in video recordings path pattern.
 
### What is the Expected behavior?

The correct folder is created for video path pattern ${TEST_INDEX} (no 'null' folder).

### What is your web application and your TestCafe test code?

 
Your website URL (or attach your complete example):

<details>
<summary>Your complete test code (or attach your test files):</summary>

 
```js
 import { Selector, ClientFunction } from 'testcafe';

fixture("Simple test")
        .page("https://devexpress.github.io/testcafe/example");

test('Zero Test', async t => {
	console.log(await t.eval(() => ({ w: window.outerWidth, h: window.outerHeight})))
	await t.expect(false).ok('this assertion will fail');
});
		

test('First Test', async t => {
    await t.setTestSpeed(1);
	
	debugger;
	
	console.log('testing console log');
	
    await t.typeText('#developer-name', 'Alex')
		.click('#submit-button');
		
		const { error } = await t.getBrowserConsoleMessages();

    await t.expect(error).eql([]);
});

test('Second test', async t => {
	
	console.log('testing console log from the second test');
	
    await t.setTestSpeed(0.5);
});
```
</details>

<details>
<summary>Your complete config file:</summary>

```
{
    "browsers": ["chrome", "firefox:headless"],
    "src": ["test.js"],
    "screenshotPath": "./screens",
    "screenshotPathPattern": "${DATE}_${TIME}/test-${TEST_INDEX}/${USERAGENT}/${FILE_INDEX}.png",
    "takeScreenshotsOnFails": true,
    "videoPath": "reports/screen-captures",
    "videoOptions": {
        "singleFile": true,
        "failedOnly": true,
        "pathPattern": "${TEST_INDEX}/${USERAGENT}/${FILE_INDEX}.mp4"
    },
    "videoEncodingOptions": {
        "r": 20,
        "aspect": "4:3"
    },
    "stopOnFirstFail": false,
    "skipJsErrors": true,
    "skipUncaughtErrors": true,
    "concurrency": 1,
    "selectorTimeout": 3000,
    "assertionTimeout": 1000,
    "pageLoadTimeout": 1000,
    "speed": 1
}
```
</details>

<details>
<summary>Your complete test report:</summary>

```
c:\Temp\!TestCafe>testcafe
Using locally installed version of TestCafe.
 Running tests in:
 - Chrome 72.0.3626 / Windows 10.0.0
 - Firefox 65.0.0 / Windows 10.0.0

 Simple test
{ w: 1366, h: 768 }
testing console log from the second test
testing console log from the second test
{ w: 1051, h: 806 }
testing console log
testing console log
 × Zero Test (screenshots: c:\Temp\!TestCafe\screens\2019-02-14_20-24-52\test-1)

   1) AssertionError: this assertion will fail: expected false to be truthy

      Browser: Chrome 72.0.3626 / Windows 10.0.0
      Screenshot: c:\Temp\!TestCafe\screens\2019-02-14_20-24-52\test-1\Chrome_72.0.3626_Windows_10.0.0\errors\1.png

          3 |fixture("Simple test")
          4 |        .page("https://devexpress.github.io/testcafe/example");
          5 |
          6 |test('Zero Test', async t => {
          7 |    console.log(await t.eval(() => ({ w: window.outerWidth, h: window.outerHeight})))
       >  8 |   await t.expect(false).ok('this assertion will fail');
          9 |});
         10 |
         11 |
         12 |test('First Test', async t => {
         13 |    await t.setTestSpeed(1);

         at ok (c:\Temp\!TestCafe\test.js:8:24)


   2) AssertionError: this assertion will fail: expected false to be truthy

      Browser: Firefox 65.0.0 / Windows 10.0.0
      Screenshot: c:\Temp\!TestCafe\screens\2019-02-14_20-24-52\test-1\Firefox_65.0.0_Windows_10.0.0\errors\1.png

          3 |fixture("Simple test")
          4 |        .page("https://devexpress.github.io/testcafe/example");
          5 |
          6 |test('Zero Test', async t => {
          7 |    console.log(await t.eval(() => ({ w: window.outerWidth, h: window.outerHeight})))
       >  8 |   await t.expect(false).ok('this assertion will fail');
          9 |});
         10 |
         11 |
         12 |test('First Test', async t => {
         13 |    await t.setTestSpeed(1);

         at ok (c:\Temp\!TestCafe\test.js:8:24)


 √ First Test
 √ Second test


 1/3 failed (14s)
```
</details>

### Steps to Reproduce:

Run testcafe with the specified config.
 
### Your Environment details:
 
* testcafe version:                    1.0.1-rc.1
* command-line arguments:   testcafe
* browser name and version:  see config
* platform and version:           Win 10

### PR DESCRIPTION
**Pull request by miherlosev**
@AndreyBelym @AlexKamaev @VasilyStrelyaev

### CODE REVIEW

File: src/notifications/warning-message.js [hunk 1]
```diff
@@ -22,5 +22,10 @@ export default {
 
     cannotFindConfigurationFile: 'Unable to find the "{path}" configuration file. Error details:\n' +
                                  '\n' +
-                                 '{err}'
+                                 '{err}',
+
+    problematicPathPatternPlaceholderForVideoRecording: '{placeholderList} path pattern placeholder{suffix} {verb} not suitable for the video recording\'s "pathPattern" option.\n' +
+                                                        '\n' +
+                                                        '{pronoun} value{suffix} will be replaced with an empty string.'
```
Comment (VasilyStrelyaev) [inline src/notifications/warning-message.js]: Dirk's version:

```js
    problematicPathPatternPlaceholderForVideoRecording: 'The {placeholderList} path pattern placeholder{suffix} cannot be applied to the recorded video.\n' +
                                                        '\n' +
                                                        'The placeholder{suffix} {verb} replaced with an empty string.'
```

where `{suffix}` is `s` and `{verb}` is either `was` or `were`.

**Comment by VasilyStrelyaev** [comment]
It was the Past tense - `was` and `were` - in the message approved by Dirk, not `are` and `is`.

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@VasilyStrelyaev FPR

**Pull request merged**